src/views/transaction-details/transaction-details.view.tsx

```diff
@@ -0,0 +1,112 @@
+import { FC } from "react";
+import { Link, Navigate, useParams } from "react-router-dom";
+
+import useTransactionDetailsStyles from "src/views/transaction-details/transaction-details.styles";
+import Card from "src/views/shared/card/card.view";
+import Header from "src/views/shared/header/header.view";
+import { ReactComponent as NewWindowIcon } from "src/assets/icons/new-window.svg";
+import { ReactComponent as SpinnerIcon } from "src/assets/icons/spinner.svg";
+import Typography from "src/views/shared/typography/typography.view";
+import { StatusText } from "src/views/activity/components/transaction-card/transaction-card.view";
+import { demoData } from "src/views/activity/demo-data";
+import { getTimeFromNow } from "src/utils/time";
+import { convertTokenAmountToFiat } from "src/utils/amounts";
+import TokenIcon from "src/views/shared/token-icon/token-icon";
+import ChainRow from "src/views/transaction-details/components/chain-row/chain-row";
+
+const TransactionDetails: FC = () => {
+  const { transactionId } = useParams();
+  const data = demoData.find((demo) => demo.id.toString() === transactionId);
+  const classes = useTransactionDetailsStyles({ status: data?.status || "initiated" });
+
+  if (!data) {
+    return Navigate({ to: "/activity", replace: true });
```

Comment:
`Navigate` is a React component, it should be instantiated properly as a React component